egs/libri_css/asr1_mono/RESULTS.md

```diff
@@ -0,0 +1,16 @@
+# Transformer results
+
+This example demonstrates usage of pre-trained LibriSpeech model
+for the decoding of [LibriCSS](https://arxiv.org/abs/2001.11482) dataset.
+
```

Comment:
Could you also briefly describe the models you used to get these results here?

Reply:
Done.